src/test/java/utils/PropertyReader.java

```diff
@@ -0,0 +1,51 @@
+package utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertyReader {
+
+    public static String getBaseUrl() {
+        return getProperty("url");
+    }
+
+    public static Browser getBrowser() {
+        return Browser.valueOf(getProperty("browser"));
+    }
+
+    public static int getDefaultWebDriverWait() {
+        return Integer.parseInt(getProperty("defaultWebDriverWaitSeconds"));
+    }
+
+    public static String getProperty(String propertyName) {
+        if (System.getProperty(propertyName) == null) {
+            return getPropertyFromFile(propertyName);
+        } else {
+            return System.getProperty(propertyName);
+        }
+    }
+
+    private static String getPropertyFromFile(String propertyName) {
+        Properties prop = new Properties();
+        InputStream input = null;
+
+        try {
+            input = new FileInputStream("src/test/resources/framework.properties");
```

Comment:
Move the path to a constant variable inside this class

Reply:
fixed